source/install/install-rhel-8.rst

```diff
@@ -98,4 +98,37 @@ If you wish to remove the Mattermost Server for any reason, you must stop the Ma
 
 	Depending on your configuration, there are several important folders in ``/opt/mattermost`` to backup. These are ``config``, ``logs``, ``plugins``, ``client/plugins``, and ``data``. We strongly recommend you back up these locations before running the ``rm`` command.
 
-You may also remove the Mattermost systemd unit file and the user/group created for running the application.
\ No newline at end of file
+You may also remove the Mattermost systemd unit file and the user/group created for running the application.
+
+Frequently asked questions
+--------------------------
+
+Can I run Mattermost without a proxy?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. However, like NGINX, in place, Mattermost binds to 443 instead of 8065. The Mattermost binary requires the correct permissions to do that binding. You must activate the ``CAP_NET_BIND_SERVICE`` capability to allow the new Mattermost binary to bind to ports lower than 1024 by running the following command:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    sudo setcap cap_net_bind_service=+ep ./mattermost/bin/mattermost
+
+.. note::
+  
+  - We don't recommend deploying Mattermost on a single server unless you have fewer than 200 users.
```

Comment:
Right now we can't really make this claim and since MM Omnibus is not limited to 200 users I would remove this item.

Maybe a note that having a proxy in front on Mattermost is highly recommended and if setup/maintenance is a worry then use MM Omnibus.